### ISSUE TITLE
player house not looted if whitelisted

### ISSUE THREAD

**Issue by SteveTownsend** [SteveTownsend/SmartHarvestSE#107]
Discord report re Rkund. Whitelisting was not causing dead body looting. Applied a fix to change the logic a little.

**Comment by SteveTownsend** [comment]
Test evidence after fix to whitelist this if mod is present
```
J:\GitHub\SmartHarvestSE\src\Data\dataCase.cpp(505): [MESSAGE] {7548} (2020-07-05 09:42:13 AM) Record Corpse To Coinage Corpse Coinage(0xd900aa03) as septims
```
```
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(1057): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) Process REFR 0xff002306 with base object Corpse Coinage/0xd900aa03
J:\GitHub\SmartHarvestSE\src\Utilities\debugs.cpp(137): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) 0xd900aa03 0x18(24) [Corpse Coinage] - typename septims
J:\GitHub\SmartHarvestSE\src\Looting\tasks.cpp(395): [VERBOSE] {24288} (2020-07-05 10:07:17 AM) SmartHarvest Corpse Coinage/0xd900aa03 for REFR 0xff002306 at distance 126.01 units
```

**Comment by SteveTownsend** [comment]
Logic reivew needed for locations "Player House" vs "owned by friendly faction".

**Comment by SteveTownsend** [comment]
The revised version should also notify that Friendly Faction locations are restricted, which it did not before since it's not a Player House but just owned by friendlies. All these checks are done the same way now, in one place before the scan even kicks off.

**Comment by SteveTownsend** [comment]
fixed in v1.6.1.0
